model/protonet.py

```diff
@@ -0,0 +1,100 @@
+import torch.nn.functional as F
+import numpy as np
+import torch
+from torch import nn
+
+class InterpolateModule(nn.Module):
+	"""
+	This is a module version of F.interpolate (rip nn.Upsampling).
+	Any arguments you give it just get passed along for the ride.
+	"""
+
+	def __init__(self, *args, **kwdargs):
+		super().__init__()
+
+		self.args = args
+		self.kwdargs = kwdargs
+
+	def forward(self, x):
+		return F.interpolate(x, *self.args, **self.kwdargs)
+
+class InterpolateModule(nn.Module):
+	"""
+	This is a module version of F.interpolate (rip nn.Upsampling).
+	Any arguments you give it just get passed along for the ride.
+	"""
+
+	def __init__(self, *args, **kwdargs):
+		super().__init__()
+
+		self.args = args
+		self.kwdargs = kwdargs
+
+	def forward(self, x):
+		return F.interpolate(x, *self.args, **self.kwdargs)
```

Comment:
It's better to remove this class and just call F.interpolate directly in the other layers/modules